src/Config/Level/CodeQualityLevel.php

```diff
@@ -100,6 +101,7 @@ final class CodeQualityLevel
      * @var array<class-string<RectorInterface>>
      */
     public const RULES = [
+        RemoveReadonlyPropertyVisibilityOnReadonlyClassRector::class,
```

Comment:
I prefer in the last for this

Reply:
Tomas said to put it anywhere I would expect it and the docblock says that the safest rules will always be at the top:

<img width="1689" height="187" alt="image" src="https://github.com/user-attachments/assets/bc467177-b5dd-41be-a3f0-cc8cea9ebe6b" />

This is an extremely safe rule so it does not belong at the end...